lizrd/train/train_utils.py

```diff
@@ -54,8 +62,18 @@ def get_model(
             vocab_size, dm, init_type=init_type, init_scale=init_scale
         ).to(first_gpu),
     )
+    embedding_layer = wrap_in_fsdp(
+        enabled=wrap_blocks_in_fsdp,
+        module=embedding_layer,
+        rank=rank,
+        param_precision=param_precision,
+        offload_params=offload_params,
+    )
 
-    layer_dict = {"attention": attention_layer_fun, "feedforward": ff_layer_fun}
+    layer_dict = {
+        "attention": attention_layer_fun,
+        "feedforward": ff_layer_fun,
+    }
```

Comment:
I know nothing is changed here, but would you be able to change it to LayerDict? So we can delete the comment below - the code itself should signify that order definitely matters here.

Reply:
It's beyond range of this PR